sale_force_invoiced/wizard/sale_make_invoice_advance.py

```diff
@@ -0,0 +1,26 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class SaleAdvancePaymentInv(models.TransientModel):
+    _inherit = "sale.advance.payment.inv"
+
+    def _check_sale_orders_fully_invoiced(
+        self,
+    ):
+        for sale_order in self.sale_order_ids:
+            if sale_order.invoice_status == "invoiced":
+                raise UserError(
+                    _(
+                        "The order %s is blocked for invoicing. "
```

Comment:
Better to use translation function arguments:

```suggestion
                        "The order %(name)s is blocked for invoicing. "...., name=sale_order.name)
```

Reply:
Done